virttest/utils_misc.py

```diff
@@ -2756,14 +2756,25 @@ def format_linux_disk(session, did, mountpoint, size, fstype="ext3"):
     for disk in disks.splitlines():
         if did in disk:
             kname = disk.split()[0]
+            list_parttable_cmd = "parted -s %s print" % kname
+            mklabel_cmd = "parted -s %s mklabel msdos" % kname
+            output = session.cmd_output(list_parttable_cmd)
+            if "Error" in output:
```

Comment:
do you mean not in.  if error in output, I think that's mean create partition failed.